lib/buffer.js

```diff
@@ -213,7 +214,7 @@ function allocate(size) {
     // Even though this is checked above, the conditional is a safety net and
```

Comment:
Looks like it has returned after a rebase =).
It's not critical, though, that could be removed later.